From 50ea8d6fca207ee4006ba8ed4d9b51935198e9e8 Mon Sep 17 00:00:00 2001
From: "Valery V. Vorotyntsev" <valery_vorotyntsev@xyratex.com>
Date: Wed, 4 Apr 2012 16:26:00 +0300
Subject: [PATCH 122/311] reduce font size of state diagram

This simple change allows the diagram to fit browser window.

Reviewed-on: https://reviewboard.clusterstor.com/r/714/
Reported-by: Carl Braganza <carl_braganza@xyratex.com>
---
 conf/confc.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/conf/confc.c b/conf/confc.c
index d4543e8..9140bc3 100644
--- a/conf/confc.c
+++ b/conf/confc.c
@@ -268,8 +268,8 @@
  *
  * @dot
  * digraph confd_states {
- *     node [fontsize=11];
- *     edge [fontsize=11];
+ *     node [fontsize=9];
+ *     edge [fontsize=9];
  *     S_INITIAL  [style=filled, fillcolor=lightgrey];
  *     S_TERMINAL [style=filled, fillcolor=lightgrey,
  *                 label="{ completion_announce() }\nS_TERMINAL"];
-- 
1.8.3.2

